src/Microsoft.Performance.SDK/Extensibility/DataCookerPath.cs

```diff
@@ -12,21 +12,13 @@ namespace Microsoft.Performance.SDK.Extensibility
     public struct DataCookerPath
         : IEquatable<DataCookerPath>
     {
-        /// <summary>
-        ///     Describes the format of a data cooker path.
-        /// </summary>
-        [Obsolete("Fully qualified data cooker paths will no longer be supported in SDK v1.0.0 release candidate 1")]
-        public static string Format => "SourceId/CookerId";
-
         /// <summary>
         ///     If the data cooker is not a source data cooker, this is the source parser Id.
         /// </summary>
-        [Obsolete("This will be removed by SDK v1.0.0 release candidate 1")]
-        public const string EmptySourceParserId = "";
+        internal const string EmptySourceParserId = "";
```

Comment:
why not `= DataCookerPathInternal.EmptySourceParserId` or just use that directly

Reply:
If it's internal, then we'd have to make internals visible to the runtime. Not something I want to do to solve this "problem".

Reply:
Makes sense. Should we add comments above the two definitions then saying this value must match the other? Just worried one day we'll decide to update the empty source parser ID and get some weird bugs.